src/features/common/Layout/Footer/index.tsx

```diff
@@ -31,28 +31,59 @@ export default function Footer() {
     setOpenModal(false);
   };
 
-  const FooterLinks = [
-    {
-      id: 1,
+  const [userLang,setUserLang] = React.useState('en');
+  React.useEffect(()=>{
+    if (typeof window !== 'undefined') {
+      let userLang = localStorage.getItem('language') || 'en';
+      setUserLang(userLang)
+    }
+  },[])
+  const FooterLinks = {
+    shop:{
+      title: ready ? t('common:shop') : '',
+      link:`https://www.thegoodshop.org/${userLang}`
+    },
+    privacyAndTerms:{
       title: ready ? t('common:privacyAndTerms') : '',
-      link: 'https://a.plant-for-the-planet.org/en/privacy-terms',
+      link: `https://a.plant-for-the-planet.org/${userLang}/privacy-terms`,
     },
-    {
-      id: 2,
+    imprint:{
       title: ready ? t('common:imprint') : '',
-      link: 'https://a.plant-for-the-planet.org/imprint',
+      link: `https://a.plant-for-the-planet.org/${userLang}/imprint`,
     },
-    {
-      id: 3,
+    contact:{
       title: ready ? t('common:contact') : '',
       link: 'mailto:support@plant-for-the-planet.org',
     },
-    {
-      id: 6,
+    downloads:{
+      title: ready ? t('common:downloads') : '',
+      link:`https://a.plant-for-the-planet.org/${userLang}/download`
+    },
+    annualReports:{
+      title: ready ? t('common:annualReports') : '',
+      link:`https://a.plant-for-the-planet.org/${userLang}/download`
+    },
+    team:{
+      title: ready ? t('common:team') : '',
+      link:`https://a.plant-for-the-planet.org/${userLang}/structure`
+    },
+    jobs:{
+      title: ready ? t('common:jobs') : '',
+      link:`https://a.plant-for-the-planet.org/${userLang}careers`
+    },
+    supportUs:{
       title: ready ? t('common:supportUs') : '',
-      link: 'https://www.plant-for-the-planet.org/en/donation',
+      link: `https://a.plant-for-the-planet.org/${userLang}/donation`,
```

Comment:
https://a.plant-for-the-planet.org/de/donation does not exist (yet?)

Reply:
revert this part.

Reply:
@sagararyal delete this item from the footer?

Reply:
ok, all the footer links now contain `pt-br` instead of `pt-BR` (by converting the local always to lowercase for all footer links).

Reply:
All these links now get redirected to `https://a.plant-for-the-planet.org/donation`  and then to `https://www.plant-for-the-planet.org/en/donation`. I think this gives us some flexibility.